src/nwb_conversion_tools/tools/spikeinterface/spikeinterface.py

```diff
@@ -966,46 +972,45 @@ def write_recording(
     assert (
         distutils.version.LooseVersion(pynwb.__version__) >= "1.3.3"
     ), "'write_recording' not supported for version < 1.3.3. Run pip install --upgrade pynwb"
-
-    assert save_path is None or nwbfile is None, "Either pass a save_path location, or nwbfile object, but not both!"
+    write_as = "raw" if write_as is None else write_as
+    compression = "gzip" if compression is None else compression
+
+    # TODO on or after August 1st, 2022, remove argument and deprecation warnings
```

Comment:
Great, I was wondering if there was a way to automatically raise an issue at that date or something to keep track of this.